.github/workflows/ci.yml

```diff
@@ -170,27 +170,19 @@ jobs:
           key: cargo-${{ matrix.config.target }}-${{ matrix.config.variant }}-${{ matrix.config.v8_enable_pointer_compression }}-${{ hashFiles('Cargo.lock', 'build.rs', 'git_submodule_status.txt') }}
           restore-keys: cargo-${{ matrix.config.target }}-${{ matrix.config.variant }}-${{ matrix.config.v8_enable_pointer_compression }}-
 
-      - name: Install and start sccache
-        shell: pwsh
+      - name: Sccache
+        uses: mozilla-actions/sccache-action@v0.0.8
+
+      - name: Start sccache
         env:
+          # Start sccache server instead of using `RUSTC_WRAPPER` environment variable (recommended by `sccache-action`).
+          # Because it cannot compile `syn` lib correctly on windows.
           SCCACHE_DIR: ${{ github.workspace }}/target/sccache
           SCCACHE_CACHE_SIZE: 128M
```

Comment:
If sccache hit rate is too low, it may be related to the cache size here.

If there's no price issue (who will pay for the bill), this value can be set to `10G` to improve the hit rate.